src/zentables/zentables.py

```diff
@@ -614,7 +614,9 @@ def _internal_pivot_table(
             ).reset_index()
 
             if margins:
-                submargin = submargin.iloc[:-1,]
+                submargin = submargin.iloc[
```

Comment:
Should this be `submargin.iloc[:-1, :]`? Looks like R syntax to me. If I wrote this then my bad lol